packages/editor-ui/src/utils/typeGuards.ts

```diff
@@ -43,3 +43,9 @@ export const isResourceMapperValue = (value: unknown): value is string | number
 export const isJSPlumbEndpointElement = (element: Node): element is HTMLElement => {
 	return 'jtk' in element && 'endpoint' in (element.jtk as object);
 };
+
+export function isDateObject(date: unknown): date is Date {
+	return (
+		!!date && Object.prototype.toString.call(date) === '[object Date]' && !isNaN(date as number)
```

Comment:
why not use `instanceof`?

Reply:
Apparently `date instanceof Date` has some limitations as described [here](https://stackoverflow.com/questions/643782/how-to-check-whether-an-object-is-a-date). It returns true for invalid date and it fails if dates are passed across boundaries (iframes). The check above is safer.

Reply:
interesting. Might be helpful to add it as a comment because it would confusing to others.